redact_test.go

```diff
@@ -8,63 +8,167 @@ import (
 )
 
 func TestSanitizeFixture(t *testing.T) {
-	gunit.Run(new(SanitizeFixture), t)
+	gunit.Run(new(SanitizeFixture), t, gunit.Options.AllSequential())
```

Comment:
This `AllSequential()` option shouldn't be necessary for a simple library like this.

Reply:
This became an issue when we began using the global map. AllSequential was the solution to the `fatal error: concurrent map read and map write` issue

Reply:
Let's move away from the mutable, package-level (global) map to support multiple, concurrent callers (like our tests!).